include/chopper/pack/pack_data.hpp

```diff
@@ -29,11 +30,14 @@ struct pack_data
     //!\brief Information about previous levels of the IBF if the algorithm is called recursively.
     previous_level previous{};
 
+    //!\brief An object collecting statistics on the way.
+    hibf_statistics::ibf * stats{};
```

Comment:
```suggestion
    hibf_statistics::ibf * stats{nullptr};
```

This is completely not const, right?